src/app/components/shared/SkipLinks/SkipLink.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from "styled-components"
```

Comment:
Should / can we push a SkipLink component back to ASC?

Reply:
Would be nice, but it has to be much more adjustable then.